be2-scala/src/test/scala/util/examples/data/TestKeyPairs.scala

```diff
@@ -0,0 +1,23 @@
+package util.examples.data
+
+import ch.epfl.pop.model.objects._
+import spray.json._
+
+final case class KeyPairWithHash(keyPair: KeyPair, pubkeyHash: Base64Data)
+
+object TestKeyPairs {
+  private val raw = {
+    val source = scala.io.Source.fromFile("../tests/data/keypair.json")
```

Comment:
I would maybe have imported scala.io.Source for clarity but since it's only used once it might not be worth it